CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1 @@
+EDGI's Code of Conduct is available here: [EDGI Code of Conduct](https://github.com/edgi-govdata-archiving/overview/blob/master/CONDUCT.md)
```

Comment:
We should just merge this PR to get it in, but I’m wondering whether we should ultimately move the actual code of conduct document here. Not sure what the real pros/cons of that are.

If we don’t move it, we should probably rename the canonical one in the overview repo to `CODE_OF_CONDUCT.md` so GitHub will pick up on it and show it in all the relevant parts of the UI.

Reply:
I was wondering about that too, and didn't do it for 2 reasons:
1. Authorship would be lost
2. We already link to that one in so many places

But I do think we should do it as part of another full repo audit at some point, along with prb removing all of the Contributing files that are just the template because it would be better to have one default contributing file.